### PR TITLE
Fix/correct formula for new_ivl in disperse_siblings

### PR DESCRIPTION
**Pull request by user1823**
In the following code, an incorrect formula for `new_ivl` would be used if the user didn't remove the value of `easy bonus` from their scheduler code after updating to v4.
https://github.com/open-spaced-repetition/fsrs4anki-helper/blob/857cba9a70adf62473c07ebc7b8f17f7d416facb/schedule/disperse_siblings.py#L49-L54

I fixed this in the current PR. However, I am not very sure about the correctness of the syntax of the code. So, please verify.

Also, in the above code, the `new_ivl` was calculated using the old formula in v4 when last rating ≠ 4. I have corrected this too.

### CODE REVIEW

File: schedule/disperse_siblings.py [hunk 1]
```diff
@@ -47,13 +47,13 @@ def get_due_range(cid, parameters, stability, due):
     revlogs = filter_revlogs(mw.col.card_stats_data(cid).revlog)
     last_due = get_last_review_date(revlogs[0])
     last_rating = revlogs[0].button_chosen
-    if last_rating == 4:
-        try:
-            new_ivl = int(round(stability * parameters['e'] * math.log(parameters['r']) / math.log(0.9)))
-        except KeyError:
-            new_ivl = int(round(9 * stability * (1 / parameters['r'] - 1)))
+    if version[0] == 4:
```
Comment (L-M-Sherlock) [inline schedule/disperse_siblings.py:50]: We need pass the version variable when calling this function.

**Review by L-M-Sherlock — CHANGES_REQUESTED** [review]

File: schedule/disperse_siblings.py (hunk 1 above)
Comment (L-M-Sherlock) [reply schedule/disperse_siblings.py:50]: Version is not a global variable.

Comment (user1823) [reply schedule/disperse_siblings.py:50]: So, how to do it?

**Comment by user1823** [comment]
With the latest commit, I have renamed "Young and Mature" in the true retention table to "Total" for conciseness.

File: schedule/disperse_siblings.py (hunk 1 above)
Comment (L-M-Sherlock) [reply schedule/disperse_siblings.py:50]: Have you tested it in your device? I will review it tomorrow.

Comment (user1823) [reply schedule/disperse_siblings.py:50]: I have not tested it because I don't exactly how to test this.

Comment (L-M-Sherlock) [reply schedule/disperse_siblings.py:50]: You can replace the code in your local add-on file.

Comment (user1823) [reply schedule/disperse_siblings.py:52]: > You can replace the code in your local add-on file.

I knew that. The problem was that I didn't know what to look while testing. But, then I decided to test it anyway. I initially got an error and then fixed it. Apart from that, it seems to work fine (but I am not sure).

**Review by L-M-Sherlock — APPROVED** [review]

**Pull request merged**